### PR TITLE
docs: add 10 cbdb docs

### PR DESCRIPTION
**Pull request by TomShawn**
Add 10 cbdb docs. These docs are validated or rewritten by @TomShawn. Please help check these docs.

- Start and Stop Cloudberry Database
- Create and Manage Cloudberry Database
- Connect to Cloudberry Database
- Basic Queries of Cloudberry Database
- Insert, Update, Delete Row Data
- Work with Transactions
- Transactional Concurrency Control
- Create and Manage Tables
- Query Performance Overview
- Update Statistics Using Analyze

---

## Change logs

> Describe your change clearly, including what problem is being solved or what document is being added or updated.

## Contributor's checklist

Here are some reminders before you submit your pull request:

* Make sure that your Pull Request has a clear title and commit message. You can take the [Git commit template](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) as a reference.
* Sign the Contributor License Agreement as prompted for your first-time contribution (*One-time setup*).
* Learn the [code contribution](https://cloudberrydb.org/contribute/code) and [doc contribution](https://cloudberrydb.org/contribute/doc) guides for better collaboration.
* Make sure that your changes deployment preview is successful.
* List your communications in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb-site/issues) or [Discussions](https://github.com/orgs/cloudberrydb/discussions) (if has or needed).
* Feel free to ask for the @cloudberrydb/doc team or other people to help review and approve.

### CODE REVIEW

**Review by tuhaihe — COMMENTED** [review]
Left some comments, FYI.

**Comment by tuhaihe** [comment]
The website deployment failed, it showed some errors on the doc links:

```
6:32:31 PM: Please check the pages of your site in the list below, and make sure you don"t reference any path that does not exist.
Note: it"s possible to ignore broken links with the "onBrokenLinks" Docusaurus configuration, and let the build pass.
6:32:31 PM: Exhaustive list of all broken links found:
6:32:31 PM: 
6:32:31 PM: - On source page path = /docs/create-and-manage-tables:
6:32:31 PM:    -> linking to /docs/sql-stmts/sql-stmt-create-table.md
6:32:31 PM: 
6:32:31 PM: - On source page path = /docs/update-stats-using-analyze:
6:32:31 PM:    -> linking to /docs/sql-stmts/sql-stmt-analyze.md
6:32:31 PM:    -> linking to /docs/db-utilities/db-util-analyzedb.md
6:32:31 PM: 
6:32:31 PM:     at throwError (/opt/build/repo/node_modules/@docusaurus/logger/lib/index.js:76:11)
6:32:31 PM:     at handleBrokenLinks (/opt/build/repo/node_modules/@docusaurus/core/lib/server/brokenLinks.js:153:47)
6:32:31 PM:     at async buildLocale (/opt/build/repo/node_modules/@docusaurus/core/lib/commands/build.js:186:5)
6:32:31 PM:     at async tryToBuildLocale (/opt/build/repo/node_modules/@docusaurus/core/lib/commands/build.js:41:20)
6:32:31 PM:     at async mapAsyncSequential (/opt/build/repo/node_modules/@docusaurus/utils/lib/jsUtils.js:34:24)
6:32:31 PM:     at async Command.build (/opt/build/repo/node_modules/@docusaurus/core/lib/commands/build.js:76:21)
```

File: i18n/zh/docusaurus-plugin-content-docs/current/basic-query-syntax.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    SELECT * FROM employees ORDER BY salary DESC;  -- 按薪水降序排列员工信息。
+    ```
+
+- 聚合函数：用于从数据集中计算统计信息，例如 `COUNT`、`SUM`、`AVG`、`MAX`、`MIN`。
+
+    ```sql
+    SELECT AVG(salary) FROM employees;  -- 计算员工的平均薪水。
+    ```
+
+- `GROUP BY`：与聚合函数一起使用，用于对结果集进行分组。
```
Comment (my-ship-it) [inline i18n/zh/docusaurus-plugin-content-docs/current/basic-query-syntax.md:35]: 用于指定分组聚合的列信息

**Review by my-ship-it — CHANGES_REQUESTED** [review]
LGTM except some comments

**Review by tuhaihe — APPROVED** [review]
LGTM.

**Review by my-ship-it — APPROVED** [review]
LGTM

**Pull request merged**